frontend/src/components/Menu/DropdownMenu.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Box,
+  Menu as ChakraMenu,
+  MenuButton as ChakraMenuButton,
+  MenuItem as ChakraMenuItem,
+  MenuItemProps,
+  MenuList as ChakraMenuList,
+  MenuListProps,
+  MenuProps,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+import { BxsChevronDown } from '~/assets/icons/BxsChevronDown'
+import { BxsChevronUp } from '~/assets/icons/BxsChevronUp'
+import {
+  DropdownMenuSize,
+  DropdownMenuVariant,
+} from '~/theme/components/DropdownMenu'
+
+export interface DropdownMenuButtonProps {
+  /**
+   * The variant of the dropdown menu.
+   */
+  variant: DropdownMenuVariant
+  /**
+   * Whether the menu has been opened
+   */
+  isActive: boolean
+  /**
+   * The size of the dropdown menu - stretched or not.
+   */
+  size: DropdownMenuSize
+  /**
+   * Menu title
+   */
+  children: string
+}
+
+/**
+ * @preconditions Must be a child of DropdownMenu component and returned
+ * using a render prop (see implementation in DropdownMenu.stories)
+ */
+export const DropdownMenuButton = ({
+  variant = 'outline',
+  children,
+  isActive,
+  size,
+}: DropdownMenuButtonProps): JSX.Element => {
+  const styles = useMultiStyleConfig('DropdownMenu', { variant, size })
+  return (
+    <ChakraMenuButton
+      sx={styles.outerBox}
+      as={Box}
+      bgColor={isActive ? 'secondary.300' : 'clear'} //  For rounded outline
+    >
+      <Box sx={styles.innerBox}>
+        {children}
+        {isActive ? <BxsChevronUp /> : <BxsChevronDown />}
```

Comment:
should we set `aria-role: presentation` and `aria-hidden: true`? as this whole element is a button, the icon is purely presentational and hiding it from keyboard events/screen readers would avoid the problem of this being accidentally tab-targeted. 

a good article on `aria-role: presentation` is [this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_presentation_role) one!

Reply:
thanks! added